split.Console/Models/Game.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace split.Console.Models
+{
+    public class Game
+    {
+        public Guid Id;
+        public string Code;
+        public GameType Type;
+        public List<Player> Players;
+        public GameState State;
+
+        public Game(GameType type)
+        {
+            Code = GenerateRoomCode();
+            Id = new Guid(Code);
```

Comment:
@rats3g and I used GUID in one project. I think C# [provides a simple way](https://msdn.microsoft.com/en-us/library/system.guid.newguid(v=vs.110).aspx)  to create GUID without resorting to generating a random code. It's `Guid g = Guid.NewGuid();`